manager/catalog.go

```diff
@@ -315,14 +317,21 @@ func readRancherCompose(relativePath string, newTemplate *model.Template) error
 }
 
 func readFile(relativePath string, fileName string) (*[]byte, error) {
-	filename, err := filepath.Abs(relativePath + "/" + fileName)
+	filePath := path.Join(relativePath, fileName)
+	filename, err := filepath.Abs(filePath)
 	if err != nil {
-		log.Errorf("Error forming path to file %s, error: %v", relativePath+"/"+fileName, err)
+		log.Errorf("Error forming path to file %s, error: %v", filePath, err)
```

Comment:
@joshwget yes we should. This was already there in the code so I missed it while adding changes